docs/security/threat-model.md

```diff
@@ -1,223 +1,303 @@
 can attempt to use MITM tooling to intercept traffic between the API server
 and admission controller webhook.
 
-**Mitigation**
+#### Mitigation
+
 Webhook mTLS authentication is used.
 
-**To do**
+#### To-do
+
 Implement mutual TLS authentication.
 Additionally, it would be possible to add a policy within the recommended
 policies section in the `kubewarden-defaults` which drops the `NET_RAW`
 capability.
 
-## Threat 9 - Attacker steals traffic from the webhook via spoofing
+### Threat 9 - Attacker steals traffic from the webhook via spoofing
+
+#### Scenario
+
 An attacker is able to redirect traffic from the API server which is intended
 for the admission controller webhook by spoofing.
 
-**Mitigation**
+#### Mitigation
+
 Webhook mTLS authentication is used.
 
-**To do**
+#### To-do
+
 Kubewarden should implement mutual TLS authentication
 
-## Threat 10 - Abusing a mutation rule to create a privileged container
+### Threat 10 - Abusing a mutation rule to create a privileged container
+
+#### Scenario
+
 An attacker is able to cause a mutating admission controller to modify a workload,
 such that it allows for privileged container creation.
 
-**Mitigation**
+#### Mitigation
+
 All rules are reviewed and tested.
 
-**To do**
+#### To-do
+
 The Kubewarden team may come up with some tests to cover the review of these rules in the future.
-In addition to the above, any change of rules during policies deployment must be carefully reviewed.
+Any change of rules during policies deployment must be carefully reviewed.
+
+### Threat 11 - Attacker deploys workloads to namespaces that are exempt from admission control
+
+#### Scenario
 
-## Threat 11 - Attacker deploys workloads to namespaces that are  exempt from admission control
 An attacker is able to deploy workloads to Kubernetes namespaces that are exempt
 from the admission controller configuration.
 
-**Mitigation**
-RBAC rights are strictly controlled
+#### Mitigation: RBAC rights are strictly controlled
+
+#### To-do
 
-**To do**
 Most of the RBAC is out of scope with respect to this decision. However, the Kubewarden team aims to:
+
 - Warn users via our docs and *suggest* some minimum RBAC to be used.
 - Provide a policy which detects RBAC changes and **maybe** block them.
 
-## Threat 12 - Block rule can be bypassed due to missing match (e.g.  missing initcontainers)
+### Threat 12 - Block rule can be bypassed due to missing match (for example, missing initcontainers)
+
+#### Scenario
+
 An attacker created a workload manifest which uses a feature of the Kubernetes
 API which is not covered by the admission controller
 
-**Mitigation**
+#### Mitigation
+
 All rules are reviewed and tested.
 
-**To do**
+#### To-do
+
 Introduce tests to cover this rule.
-As always, carefully review PRs changing the rules in the policies deployment.
+As always, you should review PRs changing the rules in the policies deployment.
```

Comment:
I would drop this TODO.

On our side we have to keep up with Kubernetes releases and review the changes they introduce. When init container have been introduced we reviewed our policies to ensure they would catch them too.

On the user side, they have to review their in place rules and make sure they are covering these new corner cases.

Reply:
Thanks.